packages/client/src/views/Home/RecordAudit.tsx

```diff
@@ -0,0 +1,578 @@
+  }
+
+  let info: ILabel = {
+    status: application?.status && finalStatus,
+    type: getCaptitalizedWord(application?.type),
+    trackingId: application?.trackingId
+  }
+
+  if (info.type === 'Birth') {
+    if (application?.brnDrn) {
+      info = {
+        ...info,
+        brn: application.brnDrn
+      }
+    }
+    info = {
+      ...info,
+      dateOfBirth: application?.dateOfBirth,
+      placeOfBirth: application?.placeOfBirth,
+      informant: informant
+    }
+  } else if (info.type === 'Death') {
+    if (application?.brnDrn) {
+      info = {
+        ...info,
+        drn: application.brnDrn
+      }
+    }
+    info = {
+      ...info,
+      dateOfDeath: application?.dateOfDeath,
+      placeOfDeath: application?.placeOfDeath,
+      informant: informant
+    }
+  }
+  return (
+    <>
+      {Object.entries(info).map(([key, value]) => {
+        return (
+          <InfoContainer id={'summary'} key={key}>
+            <KeyContainer id={`${key}`}>{KEY_LABEL[key]}</KeyContainer>
+            <ValueContainer id={`${key}_value`} value={value}>
+              {value ? (
+                key === 'dateOfBirth' || key === 'dateOfDeath' ? (
+                  moment(new Date(value)).format('MMMM DD, YYYY')
+                ) : (
+                  value
+                )
+              ) : isDownloaded ? (
+                NO_DATA_LABEL[key]
+              ) : (
+                <GreyedInfo id={`${key}_grey`} />
+              )}
+            </ValueContainer>
+          </InfoContainer>
+        )
+      })}
+    </>
+  )
+}
+
+export const ShowRecordAudit = (props: IFullProps) => {
+  const applicationId = props.match.params.applicationId
+  let application: IApplicationData | null
+  application = getSavedApplications(props)
+  const isDownloaded = application ? true : false
+  if (!isDownloaded) {
+    application = getWQApplication(props)
+  }
+
+  return (
+    <div id={'recordAudit'}>
+      <Header />
+      <Navigation deselectAllTabs={true} />
+      <BodyContainer>
+        {application ? (
+          <Content
+            title={application.name || 'No name provided'}
+            titleColor={application.name ? 'copy' : 'grey600'}
+            size={'large'}
+            icon={() => (
+              <ApplicationIcon
+                color={
+                  STATUSTOCOLOR[(application && application.status) || 'DRAFT']
+                }
+              />
+            )}
+          >
+            {getApplicationInfo(props, application, isDownloaded)}
+          </Content>
+        ) : (
+          <Query
+            query={FETCH_APPLICATION_SHORT_INFO}
+            variables={{
+              id: applicationId
+            }}
+          >
+            {({
+              loading,
+              error,
+              data
+            }: {
+              loading: any
+              data?: any
+              error?: any
+            }) => {
+              if (error) {
+                goBack(props)
```

Comment:
We are not returning anything from here. So it will return the default JSX which I think assumes that there were no errors. So `<Redirect />` might be better choice for this scenario.